### ISSUE TITLE
left/right arrows even when using a touchscreen

### ISSUE THREAD

**Issue by foxharp** [saimn/sigal#505]
The photoswipe code goes out of its way to disable the left and right arrow icons when it's known that the display device has a  touchscreen (where the assumption is that swiping left/right will be used instead).  That's okay, unless your laptop has a touchscreen.  Then, when you bring up the gallery, and you're busy using your keyboard and mouse for everything, and had sort of forgotten for a while that you have a touchscreen, you find yourself without a way to navigate to the next/previous images.  

It would be better to have the arrows always present, or, at least, for there to be an obvious way to enable them.  I'm normally pretty good at ferreting out how to tweak javascript to do what I want, and am running a customized copy of photoswipe, but I can't for the life of me figure out how to get the arrows to appear unconditionally.  (A pointer would be most welcome!)

**Comment by saimn** [comment]
That's a feature request for Photoswipe ;).
But the version in sigal is pretty old (v4.4.0), maybe worth checking their release notes first (https://github.com/dimsemenov/PhotoSwipe/releases). And I should update it at some point ...

**Comment by saimn** [comment]
I updated photoswipe to the latest version (#528), please test if it is better now.

**Comment by foxharp** [comment]
No change, I'm afraid.    I get arrows (which disappear on a short timeout, and reappear with mouse movement) on a non-touchscreen display, but no arrows at all on my laptop.   Keyboard arrow keys work on both.  Both are running the latest Google Chrome.

I've kind of gotten used to it.  :-)

**Comment by saimn** [comment]
Then I guess it's something to report on photoswipe side if you want to ;)

**Comment by saimn** [comment]
Yes currently you would need to test from the git version. I will try to do a release "soon".
